cdk/lib/newswires.ts

```diff
@@ -469,5 +478,31 @@ export class Newswires extends GuStack {
 		newswiresApp.autoScalingGroup.connections.addSecurityGroup(
 			database.accessSecurityGroup,
 		);
+
+		if (this.stage === 'PROD') {
```

Comment:
I wonder if it's changing this to `this.stage !== 'CODE'` or `['TEST', 'PROD'].includes(this.stage)` so that it shows up in the snapshots?

Reply:
Yeah, I was thinking about the snapshots too. I've added a test check as well so we can see the cloudformation in the snapshot